docker/0.maven-settings.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Regarding this template, maybe it would be necessary to specify somewhere else how to instantiate it with specific values, I mean, which libraries are needed, which version, the plugins required...